### PR TITLE
Feature/mc 12216 delete networking resources

### PR DESCRIPTION
**Pull request by m-orfanos**
Kubernetes delete networking resources documentation

### CODE REVIEW

File: source/includes/kubernetes/_k8_ingresses.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ curl -X GET \
         "selfLink": "/apis/extensions/v1beta1/namespaces/auth/ingresses/cm-acme-http-solver-75png",
         "uid": "48720f48-f2bc-45fc-95c5-60cae8ffe11e"
       },
-      "spec":{
-        "rules":[]
+      "spec": {
+        "rules": []
```
Comment (m-orfanos) [inline source/includes/kubernetes/_k8_ingresses.md]: The vscode formatter fixed a bunch of whitespace 🤷‍♂️

File: source/includes/kubernetes/_k8_ingresses.md [hunk 2]
```diff
@@ -201,22 +202,50 @@ curl -X POST \
...
-| `metadata.name` <br/>_string_              | The name of the ingress.                                                |
-| `spec`<br/>_object_                        | The specification used to create and run the ingress.                   |
-| `spec.rules`<br/>_object_                  | The list of host rules used to configure the ingress.                   |
+| Required Attributes           | &nbsp;                                                |
+| ----------------------------- | ----------------------------------------------------- |
+| `apiVersion` <br/> _string_   | The api version (versioned schema) of the ingress.    |
+| `metadata` <br/>_object_      | The metadata of the ingress.                          |
+| `metadata.name` <br/>_string_ | The name of the ingress.                              |
+| `spec`<br/>_object_           | The specification used to create and run the ingress. |
+| `spec.rules`<br/>_object_     | The list of host rules used to configure the ingress. |
```
Comment (m-orfanos) [inline source/includes/kubernetes/_k8_ingresses.md:211]: The vscode formatter fixed a bunch of whitespace issues

File: source/includes/kubernetes/_k8_services.md [hunk 3]
```diff
@@ -149,25 +150,44 @@ curl -X POST \
...
 
 | Attributes                 | &nbsp;                                           |
----------------------------- | -------------------------------------------------|
-| `taskId` <br/>*string*     | The id corresponding to the create service task. |
-| `taskStatus` <br/>*string* | The status of the operation.                     |
+| -------------------------- | ------------------------------------------------ |
+| `taskId` <br/>_string_     | The id corresponding to the create service task. |
+| `taskStatus` <br/>_string_ | The status of the operation.                     |
+
+<!-------------------- DELETE A SERVICE -------------------->
```
Comment (m-orfanos) [inline source/includes/kubernetes/_k8_services.md:171]: new content here

File: source/includes/kubernetes_extension/_k8_ingresses.md [hunk 4]
```diff
@@ -213,22 +214,54 @@ Create an ingress in a given [environment](#administration-environments).
...
----------------------------- | -------------------------------------------------|
-| `taskId` <br/>*string*     | The id corresponding to the create ingress task. |
-| `taskStatus` <br/>*string* | The status of the operation.                     |
+| -------------------------- | ------------------------------------------------ |
+| `taskId` <br/>_string_     | The id corresponding to the create ingress task. |
+| `taskStatus` <br/>_string_ | The status of the operation.                     |
+
+<!-------------------- DELETE AN INGRESS -------------------->
+
+##### Delete an ingress
```
Comment (m-orfanos) [inline source/includes/kubernetes_extension/_k8_ingresses.md:239]: new content here

File: source/includes/kubernetes_extension/_k8_services.md [hunk 5]
```diff
@@ -161,26 +161,48 @@ Create a service in a given [environment](#administration-environments).
...
----------------------------- | -------------------------------------------------|
-| `taskId` <br/>*string*     | The id corresponding to the create service task. |
-| `taskStatus` <br/>*string* | The status of the operation.                     |
+| -------------------------- | ------------------------------------------------ |
+| `taskId` <br/>_string_     | The id corresponding to the create service task. |
+| `taskStatus` <br/>_string_ | The status of the operation.                     |
+
+<!-------------------- DELETE A SERVICE -------------------->
+
+##### Delete a service
```
Comment (m-orfanos) [inline source/includes/kubernetes_extension/_k8_services.md:189]: new content here

File: source/includes/kubernetes/_k8_ingresses.md [hunk 6]
```diff
@@ -201,22 +202,50 @@ curl -X POST \
...
 
 | Attributes                 | &nbsp;                                           |
----------------------------- | -------------------------------------------------|
-| `taskId` <br/>*string*     | The id corresponding to the create ingress task. |
-| `taskStatus` <br/>*string* | The status of the operation.                     |
+| -------------------------- | ------------------------------------------------ |
+| `taskId` <br/>_string_     | The id corresponding to the create ingress task. |
+| `taskStatus` <br/>_string_ | The status of the operation.                     |
+
+<!-------------------- DELETE AN INGRESS -------------------->
```
Comment (m-orfanos) [inline source/includes/kubernetes/_k8_ingresses.md:225]: new content here

File: source/includes/kubernetes_extension/_k8_ingresses.md [hunk 7]
```diff
@@ -213,22 +214,54 @@ Create an ingress in a given [environment](#administration-environments).
...
+}
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/ingresses/:id?cluster_id=:cluster_id</code>
+
+Delete an ingress from a given [environment](#administration-environments).
+
+| Required                   | &nbsp;                                             |
+| -------------------------- | -------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to get the ingress. |
```
Comment (Shabirmean) [inline source/includes/kubernetes_extension/_k8_ingresses.md:262]: Maybe:
`The id of the cluster where the target ingress is setup.`

File: source/includes/kubernetes_extension/_k8_services.md [hunk 8]
```diff
@@ -161,26 +161,48 @@ Create a service in a given [environment](#administration-environments).
...
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/services/test-service/default?cluster_id=test-cluster"
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/services/:id?cluster_id=:cluster_id</code>
+
+Delete a service from a given [environment](#administration-environments).
+
+| Required                   | &nbsp;                                             |
+| -------------------------- | -------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to get the service. |
```
Comment (Shabirmean) [inline source/includes/kubernetes_extension/_k8_services.md:203]: Maybe:
`The id of the cluster where the target service is setup.`

**Review by Shabirmean — APPROVED** [review]
🚢

**Review by jasondias9 — APPROVED** [review]

**Pull request merged**